loader.py

```diff
@@ -234,16 +241,25 @@ def main(args):
                     auth=(config.neo4j_user, config.neo4j_password),
                     encrypted=False
                 )
-
+                if config.database_type == "memgraph":
+                    mg_host = config.neo4j_uri.replace("bolt://", "").split(":")[0]
```

Comment:
I suggest save config.neo4j_uri.replace("bolt://", "").split(":") as a variable, so next line doesn't need to  do it again.